mattermost-helm/charts/mattermost-push-proxy/values.yaml

```diff
@@ -0,0 +1,37 @@
+# Default values for mattermost-push-proxy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: brunzer/mattermost-push-proxy
```

Comment:
Can we change this to mattermost/mattermost-push-proxy? I know that doesn't exist yet, but after this is merged I will make that happen.

Reply:
Ah right! I meant to switch that to mattermost but slipped my mind. I'll have a new commit up in a min